google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/model/JobJsonProtocol.scala

```diff
@@ -70,18 +70,35 @@ object Job {
  * @see [[https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#jobconfiguration BigQuery reference]]
  *
  * @param load configures a load job
+ * @param labels the labels associated with this job
  */
-final case class JobConfiguration(load: Option[JobConfigurationLoad]) {
+final case class JobConfiguration(load: Option[JobConfigurationLoad], labels: Option[Map[String, String]]) {
```

Comment:
We could make this constructor private to avoid breaking change, I guess?  Not a big deal.

Reply:
Not sure if that would help? I think we need to add a constructor with the old signature. I should have marked all of these constructors private to begin with...

Reply:
Let's make it private and just add the MimaException. It won't be bin compat but since you added the factory method it should be fine for the Scala DSL at least.

Reply:
Sounds good. While I'm at it, can I mark all the other case class constructors private?

Reply:
Sounds good.

Reply:
Done.